src/components/utils/src/threads/thread_qt.cc

```diff
@@ -126,6 +126,11 @@ Thread::Thread(const char* name, ThreadDelegate* delegate, QObject* parent)
     , thread_command_(kThreadCommandNone)
     , thread_state_(kThreadStateNone) {
   qRegisterMetaType<QThread*>("QThread*");
+  // The number of the threads was set to the number
+  // of threads used in SDL according to ListOfThreads.
+  // This is done in order to prevent the blocking
+  // of thread at start due to insufficient pool size.
+  QThreadPool::globalInstance()->setMaxThreadCount(45);
```

Comment:
@vlantonov Where did we get this number `45`? Why is 45 and not different number?